java/gast_to_code/gast_to_code_java.py

```diff
@@ -52,10 +52,17 @@ def handle_subscript(gast):
         pass
 
     def handle_name(gast):
-        return gast["value"]
+        ''' 
+        NOTE: some places store {"type": "name", "value": "s"} while others have
+        {"type": "name", "id" : "s"} in gAST but both get routed to this func.
+        We may want to re-evaluate gAST structure regarding funcs and vars
```

Comment:
Can we create a taiga ticket for this? Since both have the same type it seems like we should stick to either `id` or `name`

Reply:
+1